pkg/vm/vm_core.go

```diff
@@ -1,18 +1,32 @@
 package vm
 
 import (
-	"errors"
 	"fmt"
 
 	"github.com/lambdaclass/cairo-vm.go/pkg/builtins"
 	"github.com/lambdaclass/cairo-vm.go/pkg/lambdaworks"
 	"github.com/lambdaclass/cairo-vm.go/pkg/vm/memory"
+	"github.com/pkg/errors"
 )
 
 type VirtualMachineError struct {
 	Msg string
 }
 
+var ErrWrongInstructionEncoding = errors.New("wrong instruction encoding")
+var ErrFailedToFetchInstruction = errors.New("failed to fetch instruction in address")
+var ErrNoRelocationFoundSegment = errors.New("no relocation found for execution segment")
+var ErrTraceNotRelocated = errors.New("trace not relocated")
+var ErrUnconstraintRes = errors.New("UnconstrainedRes")
+var ErrDiffAssertValues = errors.New("diff assert values")
+var ErrCantWriteReturnPC = errors.New("can't write return pc")
+var ErrCantWriteReturnFP = errors.New("can't write return fp")
+var ErrFailedToComputeOperandAddress = errors.New("failed to compute operand address")
+var ErrNotRelocatableValue = errors.New("not a relocatable value")
+var ErrNotFeltValue = errors.New("not a felts value")
+var ErrComputeResRelocatableMul = errors.New("compute res relocatable mul")
+var UnconstrainedResAssertEq = errors.New("UnconstrainedResAssertEq")
```

Comment:
I think it could be better if all error messages are capitalized to keep consistency